modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -194,6 +194,23 @@ definition {
 		}
 	}
 
+	@summary = "Filter the selected element"
```

Comment:
Hi @juliosilveira77, this summary isn't so clear. Maybe it should be something like "Filter the MDF Listing by the selected option". What do you think?